### ISSUE TITLE
Direct Card Payments API Update

### ISSUE THREAD

**Issue by Kantonov** [zotapay/php-sdk#8]
### Feature Request
We've changed the API for direct card payments, which must be reflected across SDKs. The documentation can be found here:
https://38fb25f8.api-docs-182.pages.dev/deposit/1.0/#card-payment-integration-2

Also in swagger format:
https://38fb25f8.api-docs-182.pages.dev/swagger/1.0/

@atanasantonov, please contact me at kantonov@zota.com, so I can provide you test credentials and setup for this, also if you have any questions.

@alexbhr please confirm, thanks.

**Comment by atanasantonov** [comment]
@Kantonov could you give some hint please if possible what is changed?

**Comment by Kantonov** [comment]
@atanasantonov yes. Up until now there was one card request containing card data. This changes as follows:
1. There is one normal deposit request, without card data, send to a new endpoint, new path.
2. The responded URL is the one where another request must be sent containing only card data.

To the SDK users this should be implemented seamlessly. So they can provide one payload (set of parameters), and we make those 2 requests happen in the background. At the end just returning the final result, the response from the second request, given everything went smoothly without errors.

**Comment by alexbhr** [comment]
@atanasantonov can this be addressed so we can ship an updated SDK?

**Comment by atanasantonov** [comment]
@alexbhr i have dropped an email on 27.07 becasue there was issues and as per your email from 06.08 waiting for @Kantonov to be back from vacation to bring light.  The email as follows:

Notes to mention:

1. There is little mismatch between the documentation and Swagger for Deposit Card - In Swagger the Deposit Card Flow body contains the card data.
2. No matter what card is used (Test Cards V1 / V2) without 3D secure the response is some HTML with the message We're sorry but payment-page doesn't work properly without JavaScript enabled. Please enable it to continue.
3. We can login as SDKexplorer in https://dashboard.zota-sandbox.com/ but we cant see the test orders there.

Questions:

1. What it means the response We're sorry but payment-page doesn't work properly without JavaScript enabled. Please enable it to continue.?
2. Is it a way we can see the orders in the Sandbox dashboard?

**Comment by Kantonov** [comment]
@atanasantonov sorry for the delay, I've responded via email, let me know if everything is answered 🙏

**Comment by atanasantonov** [comment]
@Kantonov ok, thanks. I haven't received anything and the reply it's probably to Milen. @gonzomir can you confirm please?

**Comment by gonzomir** [comment]
@atanasantonov That comment above is from Aug 30, my email is from Sep 5 and I haven't received anything yet.

**Comment by atanasantonov** [comment]
@gonzomir ouch for some reason there was an unread email in my Github folder and I clicked the link to the issue. And because we are expecting an answer totally missed the date.. Anyway it will be notification to others.

**Comment by Kantonov** [comment]
@gonzomir @atanasantonov, hope you received my email from today (3 Oct) regarding the 400 bad request issue 🙏
